deel-3-webpagina/index.html

```diff
@@ -7,10 +7,25 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
+    <nav><h2 class="analytica">ANALYTICA</h2>
+        <div class="links">
+            <a class="about-us" href="#">About us</a>
```

Comment:
In de basis is dit ook een manier om navigatie knoppen te maken. Maar gebruikelijker is om hiervoor een UL met list items toe te passen. Die gaan zich dan automatisch al als een groepje gedragen en scheelt het aanmaken van al deze classes.

Reply:
Ja ik had inderdaad (ook vanuit les 4) al begrepen dat ik dit niet op de beste manier gedaan had.